app/controllers/v1/interventions/predefined_participants_controller.rb

```diff
@@ -37,10 +37,20 @@ def update
     render json: serialized_response(predefined_user)
   end
 
+  def destroy
+    authorize! :read, Intervention
+    authorize! :read, intervention_load
+    return head :forbidden unless intervention_load.ability_to_update_for?(current_v1_user)
+
+    V1::Intervention::PredefinedParticipants::UpdateService.call(intervention_load, predefined_participant, { active: false })
```

Comment:
I'm not sure if we need to run this service to update one parameter 🤔, but I think it can stay